### DISCUSSION TITLE
Export as url maximum support

### DISCUSSION THREAD

**Discussion by MorganReid** [jgraph/drawio#3872] [Q&A]
I self-deploy a drawio service based on `jgraph/drawio.`
I drew a picture, selected `File->Export as->URL`, and pasted the URL into the wiki. it can be automatically render it into a picture. This function is very useful.
The question now is how much can this picture support? Is it related to the type of storage? Can I set it myself? I used `xml` storage and measured a picture of about 5M. It can be exported as url. If it exceeds the limit, it will prompt
![image](https://github.com/jgraph/drawio/assets/52235796/f67d7906-5332-414a-b14a-22787f702012)

**Comment by davidjgraph** [comment]
Browsers have a limit of the size of URL they can support. There's no point creating a URL that browsers don't support.
